carl-repo.json

```diff
@@ -9,11 +9,11 @@
 			"files": [
 				{
 					"path": "init.lua",
-					"url": "https://raw.githubusercontent.com/ghostdevv/cc-carl/main/packages/client/init.lua"
+					"url": "packages/client/init.lua"
```

Comment:
With this, the reason it's a URL rather than using the repo url is because the repo definition might be hosted in a different place than files, I think we should keep that behaviour

Reply:
Full URLs still work, its just relative URLs are allowed as well now. This allows the same repository definition to work no matter what branch it is on.